packages/atlas/src/components/NftTile/NftTile.tsx

```diff
@@ -17,7 +17,7 @@ export type Member = {
 }
 
 export type NftTileProps = {
-  nftStatus?: 'idle' | 'on-sale' | 'auction'
+  status?: 'idle' | 'on-sale' | 'auction'
```

Comment:
I'm aware that it's not your change, but I'm confused here. What means `on-sale`? Is it `buy-now`? If yes, can we make this consistent with changes in `useNft`?

Reply:
yes `on-sale` means that you can buy it now

Reply:
hmm, so I think `buy-now` will be a much better name.

Reply:
I'm not sure if I understand. So, you're saying that `on-sale` means either `TransactionalStatusBuyNow ` or `TransactionalStatusAuction` **but with buy now price set**, right? 
If yes, this is very confusing. Maybe we could differentiate this even more - status could be `idle` or `only-buy-now` or `auction-buy-now` or `auction`?

Reply:
I think we should reflect transactional status. So `idle`, `auction`, `buy-now`, I think it's quite clear. Potentially we could split auction into separate auction types if we wanted but up to discussion

Reply:
I wouldn't split auction but add a type inside of 'open' | 'english', did this in my PR with the new changes